config/config.go

```diff
@@ -960,6 +961,13 @@ func (t *OpenTracing) ToTracingConfig() *tracing.Configuration {
 }
 
 func init() {
+	initByLDFlags(versioninfo.CommunityEdition, checkBeforeDropLDFlag)
+}
+
+func initByLDFlags(edition, checkBeforeDropLDFlag string) {
+	if edition != versioninfo.CommunityEdition {
```

Comment:
When will we enter this logic? It seems the arg of `edition` is `versioninfo.CommunityEdition`.

Reply:
This is for unit testing.

Reply:
Will be set from Makefile, in our enterprise release.

Reply:
![WeChatWorkScreenshot_c9cfee20-3c76-44a5-8169-dfc85378cafd](https://user-images.githubusercontent.com/4242506/87497023-45abe800-c687-11ea-96c0-ad9839c601ef.png)
It seems we don't call `initByLDFlags`. Maybe the argument was passed incorrectly.

Reply:
Sorry for typo, it seems that my Manual test method has a problem. I will fix it.